pipeline/build-stage.yml

```
@@ -0,0 +1,184 @@
+            trivy version
+            gitleaks version
+            
+      # Run GitLeaks secret scan
+      - task: Bash@3
+        displayName: 'GitLeaks Secret Scan'
+        inputs:
+          targetType: 'inline'
+          script: |
+            echo "🔐 Running GitLeaks secret detection..."
+            ./scripts/dev-security-scan.sh || exit_code=$?
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)